src/modules/appointments/appointments.resolver.ts

```diff
@@ -19,6 +19,7 @@ import { RoomsService } from '../rooms/rooms.service';
 import { CanPass } from 'src/utils/canpass.decorator';
 import { Roles } from 'src/types/enums/user-roles.enum';
 import { EditAppointmentInput } from './dto/edit-appointment.input';
+import { ScheduledAppointment } from './entities/scheduled-appointment.entity';
 
 @Resolver('Appointment')
```

Comment:
```suggestion
@Resolver('ScheduledAppointment')
```

name does not match the type in the schema

Reply:
fixed